changelogs/8.15.asciidoc

```diff
@@ -36,3 +36,4 @@ https://github.com/elastic/apm-server/compare/v8.14.3\...v8.15.0[View commits]
 - Support self-instrumentation when in managed mode by getting tracing configs via reloader {pull}13514[13514] {pull}13653[13653] {pull}13691[13691] {pull}13790[13790]
 - Add mapping for OpenTelemetry attribute `messaging.destination.name` to derive `service.target` correctly {pull}13472[13472]
 - APM Server now automatically retries document-level 429s from Elasticsearch to avoid dropping data. `output.elasticsearch.max_retries` now controls both request-level and document-level retries, and defaults to `3`. {pull}13620[13620]
+- Fix memory limit fallback check bytes to GB conversion {pull}13838[13838]
```

Comment:
This section is for 8.15.0. Section for 8.15.1 is not ready yet. If this fix is planned 8.15.1, let's keep this PR open until 8.15.0 release and we have a 8.15.1 section in the changelog for easier backport.

Reply:
I always forget about this: changelogs in release notes for patch releases are usually added right before release, instead of added at PR merge time unlike for usual PRs. If you could move the changelog entry to PR description, the release manager would pick it up at 8.15.1 release.

Reply:
Sure, thank you for raising this, will update it.

Reply:
updated the changelogs file